package.json

```diff
@@ -86,18 +90,11 @@
     "@openmrs/esm-api": "^1.3.0",
     "@openmrs/esm-module-config": "^1.2.0",
     "@openmrs/esm-patient-chart-widgets": "^1.0.2",
-    "@types/lodash-es": "^4.17.3",
-    "@types/react-router": "^5.1.4",
+    "@openmrs/esm-root-config": "^1.2.0",
```

Comment:
I reckon we should be able to get rid of this dependency for all these PRs now?

Reply:
Or should we do that later?

Reply:
Right now we cannot get rid of it. It transports the helper functions.

We'd need to find a new location for the two helper functions (or just copy them).